### DISCUSSION TITLE
Modify the min-size-to-set-rival option so that they cannot receive rivalries either.

### DISCUSSION THREAD

**Discussion by Red-73** [RoinujNosde/SimpleClans#199] [Ideas]
The **min-size-to-set-rival** option also applies to the reception of rivalries (that clans with a number less than that established cannot receive rivalry).

**Comment by RoinujNosde** [comment]
Are you sure?
This option only applies to the clan issuing the command.

**Comment by Red-73** [comment]
I think it is better that the option is for the reception of rivalry. In other words, a clan with a number of members less than that cannot receive a rivalry. I see a greater use of this option since it can prevent newly formed clans from immediately receiving a rivalry.

**Comment by RoinujNosde** [comment]
Is it a disadvantage for new clans to receive rivalries? Why?

**Comment by Red-73** [comment]
Well, of course it depends on how each one uses the plugin and how they establish rivalries on their server. But I feel that it would be better if clans with fewer players cannot receive the rivalries until they reach that number.
